manager/templates/default/css/index.css

```diff
@@ -21720,17 +21720,72 @@ body.x-body-masked .x-window-plain .x-window-mc {
           flex-wrap: nowrap;
```

Comment:
Please don't commit the compiled CSS. The integrator who merges the PR will compile the final CSS.